CONTRIBUTING.md

```diff
@@ -2,14 +2,14 @@
 If you're interested in contributing to this project, here are a few ways to do so:
 
 ## Bug fixes
-* If you find a bug, please first report it using [Github issues](https://github.com/flame-engine/oxygen/issues/new).
+* If you find a bug, please first check if it is already reported (duplicate issues will be closed), otherwise report it using [Github issues](https://github.com/flame-engine/oxygen/issues/new).
 * Issues that have already been identified as a bug will be labelled "bug".
 * If you'd like to submit a fix for a bug, send a [Pull Request](https://guides.github.com/activities/forking/#making-a-pull-request) from your own fork, also read the [How To](#how-to) and [Development Guidelines](#development-guidelines).
 * Include a test that isolates the bug and verifies that it was fixed.
 * Also update the example and documentation if necessary.
 
 ## New Features
-* If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new [Github issues](https://github.com/flame-engine/oxygen/issues/new).
+* If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new [Github issues](https://github.com/flame-engine/oxygen/issues/new). If it is a bigger change, you can also get some initial thoughts about it in our [discord](https://discord.gg/JUwwvNryDz).
```

Comment:
I wonder if discussions is a better choice than an issue

Reply:
issues -> issue

Reply:
discord chat maybe? Is Discord well known outside of gamer circles?

Reply:
I decided on `discord server`